src/network/index.js

```diff
@@ -32,8 +32,8 @@ class Network {
         // todo: just use rejectUnauthorized: false on transport-https, extend it
         // process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
 
-        this.web3bridge = new Web3Bridge(this.ctx);
-        await this.web3bridge.start(); // todo: do we need await?
+        this.blockchain = createBlockchain(this.ctx.basepath);
```

Comment:
As soon as we get rid of the `createBlockchain` function we'll be able to import the blockchain client directly from the module. The idea is to ultimately get rid of the `ctx` completely, since it is what makes all the modules coupled.

In other words, let's get rid of the `createBlockchain` function and import blockchain client directly everywhere we use it.